### DISCUSSION TITLE
请问多wan口怎么路由

### DISCUSSION THREAD

**Discussion by langziyang** [istoreos/istoreos#1457] [Q&A]
一条电信，在wan口PPPOE拨号，
一个移动，在wan1口，静态IP

现在有一个要求就是当我访问 10.xx.xx.xx这个IP时(内网)，要求走wan1口的移动，其它的就是wan口的电信。

现在的问题是默认设置下所有都走移动了

**Comment by w9315273** [comment]
你访问内网，关WAN口什么事？

**Reply by langziyang** [reply]
需要通过移动的线路去访问这个ip所在的内网，所以需要让访问这个ip的时候通过移动线路才能打开

**Reply by w9315273** [reply]
啥意思 是说在公网上去访问你这个内网？

**Reply by langziyang** [reply]
是的，移动的政务内网

**Reply by w9315273** [reply]
你还是没表达清楚问题
从目前提供的信息来看 你在公网 访问移动的WAN IP 这有什么问题 你不输入其他WAN的IP就可以了

**Reply by w9315273** [reply]
如果你打算在本机上访问10.0.0.0 且只能访问10.0.0.0的话 那就修改路由表
没猜错的话 你移动WAN IP也是10.0.0.0这个段的

**Reply by jjm2473** [reply]
1. wan1不要设置默认路由，wan需要开启默认路由（一般默认开了）
2. 在“网络”-“路由”添加你需要访问的目的ip（段）的路由，下一跳设置为wan1的网关ip，并且把这个路由关联到wan1

如果还有问题，执行`route -n`看看

**Comment by jjm2473** [comment]
这个问题不属于分流，纯粹就是不了解路由导致的。真正的分流是多个wan口都是用来连接外网的。
